core/src/test/scala/integration/kafka/api/TransactionsBounceTest.scala

```diff
@@ -103,7 +117,7 @@ class TransactionsBounceTest extends KafkaServerTestHarness {
             !shouldAbort), new ErrorLoggingCallback(outputTopic, record.key, record.value, true))
         }
         trace(s"Sent ${records.size} messages. Committing offsets.")
-        producer.sendOffsetsToTransaction(TestUtils.consumerPositions(consumer).asJava, consumerGroup)
+        commit(producer, consumerGroup, consumer)
```

Comment:
Where is this function defined?

Reply:
It's defined as a passed in parameter for `testBrokerFailure`